### DISCUSSION TITLE
Show data by user

### DISCUSSION THREAD

**Discussion by rvmadeit** [nocodb/nocodb#6773] [Q&A]
Hi, 

I'm new to nocodb, I already configure my database and data but for one of my table I want the data to be view only by the user assign to it if it's possible.

**Comment by dstala** [comment]
https://docs.nocodb.com/data-sources/actions-on-data-sources/#ui-access-control

**Comment by rvmadeit** [comment]
Yes, but we can’t display by user only by role. I’m trying to do this, if user a is connected he can only view task assign to him not all task in this table.
